README.md

```diff
@@ -15,3 +16,9 @@ optional arguments:
                         account
   --public              toggle to make this gist available publicly
 ```
+
+
+## Demo usage
+![Demo of gistup](https://raw.githubusercontent.com/tuladhar/gistup/master/demo.gif)
```

Comment:
The demo is showing up broken in README. To fix, please use relative path. See the suggested change.
```suggestion
![Demo of gistup](demo.gif)
```

Reply:
Ah, my bad! Added requested changes.